plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipmentDropChance.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.entity.Mob;
+import org.bukkit.inventory.EntityEquipment;
+
+public class EntityEquipmentDropChance extends EntityProperty<MapTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name equipment_drop_chance
+    // @input MapTag
+    // @description
+    // Controls the chance of each piece of equipment dropping when the entity dies.
+    // Specifying a drop chance of 0 will prevent the item from dropping, a drop chance of 1 will always drop the item if killed by a player, and a drop chance of higher than 1 will always drop the item no matter what the entity was killed by.

```

Comment:
Should specify what exactly _is_ the map here (e.g. map of equipment slots to drop chances).
Also nitpick but I'd remove `Specifying` to make it a bit nicer for the tag.